apps/emqx_bridge/test/emqx_bridge_api_SUITE.erl

```diff
@@ -999,6 +1000,8 @@ t_reset_bridges(Config) ->
     {ok, 200, []} = request_json(get, uri(["bridges"]), Config).
 
 t_with_redact_update(Config) ->
+    ok = snabbkaffe:start_trace(),
```

Comment:
It's better to use `?check_trace` macro.

Reply:
I'll fix it in a follow up.

Reply:
Actually, we still have to call `snabbkaffe:stop()` when using that macro, right?